package.json

```diff
@@ -51,6 +51,7 @@
     "mapbox-gl": "^0.12.0"
   },
   "dependencies": {
+    "event-emitter": "^0.3.4",
```

Comment:
could we use [events](https://www.npmjs.com/package/events) such as to be the same api as node?